accord-core/src/test/java/accord/burn/BurnTest.java

```diff
@@ -354,6 +418,7 @@ public static void main(String[] args)
         }
     }
 
+    @Timeout(value = 3, unit = TimeUnit.MINUTES)
```

Comment:
this is so the test times out rather than the build... when while true loops are found the build times out due to no progress... felt it was best to have the test timeout...

One issue with this is that the seed will be lost... we don't depend on `org.awaitility` like C* does, so could also add that test dependency and update the `run(seed)` method to do this timeout and log the seed on failure.